benchexec/resources.py

```diff
@@ -1,9 +1,4 @@
-# This file is part of BenchExec, a framework for reliable benchmarking:
-# https://github.com/sosy-lab/benchexec
-#
-# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
-#
-# SPDX-License-Identifier: Apache-2.0
+#resources_new.py
```

Comment:
Please restore the original comment with the copyright information.

Reply:
Done